src/Button/StatefulButton.tsx

```diff
@@ -137,15 +134,15 @@ export default class StatefulButton extends React.PureComponent<StatefulButtonPr
     this.timeoutId = setTimeout(() => {
       this.timeoutId = null;
       this.doResetInternalState();
-    }, this.getProps().timerMillis);
+    }, this.props.timerMillis);
```

Comment:
uhm 😕 
no need to cast to `DefaultedProps` here?

Reply:
apparently not, I'll double check

Reply:
it's not needed because `setTimeout` accepts `number | undefined`